### PR TITLE
UIIN-2296 Align the module with API breaking change (browse)

### PR DESCRIPTION
**Pull request by usavkov-epam**
## Purpose

Update `browse` interface and align related changes
https://issues.folio.org/browse/UIIN-2296
## Approach

- update interfaces versions;
- update search param for `subjects` browse;
- update formatter to get appropriate field from content data (`value` instead of `subject`);
- update unit tests

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

**Review by mkuklis — APPROVED** [review]

**Comment by usavkov-epam** [comment]
@folio-org/fe-tl-reviewers please, review this PR, thanks!

File: package.json [hunk 1]
```diff
@@ -45,6 +45,7 @@
       }
     ],
     "okapiInterfaces": {
+      "browse": "0.6 1.0",
       "inventory": "10.10 11.0 12.0",
```
Comment (Terala-Priyanka) [inline package.json:49]: Shouldn't you add 13.0 for inventory.

**Review by Terala-Priyanka — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (usavkov-epam) [reply package.json:49]: This PR is only about "Browse". The rest part relates to https://issues.folio.org/browse/UIIN-2288

Comment (Terala-Priyanka) [reply package.json:49]: ok Thank you

**Review by Terala-Priyanka — APPROVED** [review]

**Pull request merged**